addon/templates/components/editable-field.hbs

```diff
@@ -1,14 +1,26 @@
 <span class="content">
   {{#unless isEditing}}
-    <span onclick={{perform edit}} class="clickable editable">
+    <span class="clickable">
```

Comment:
This span is no longer `clickable` since the action has moved down.

Reply:
The other now clickable things will need this class applied or they may not work in some browsers.